test/integration/watcher.js

```diff
@@ -101,7 +103,7 @@ test('watcher does not rerun test files when they write snapshot files', t => {
 				killed = true;
 			}, 500);
 		} else if (passedFirst && !killed) {
-			t.is(buffer.replace(/\s/g, ''), '');
+			t.is(buffer.replace(/\s/g, '').replace(END_MESSAGE, ''), '');
```

Comment:
It'd be clearer if you replace whitespace from `END_MESSAGE` here rather than at the top, since the string assignment looks so much like the one in `lib/watcher.js`.